app/src/main/java/org/wikipedia/travel/MainPlannerFragment.java

```diff
@@ -255,15 +300,28 @@ public void success(List<Trip> list) {
         });
     }
 
+    public void updateUserDestinationList() {
+        CallbackTask.execute(() -> DestinationDbHelper.getInstance().getAllLists(), new CallbackTask.DefaultCallback <List <Trip>>() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['<' is preceded with whitespace.](https://app.codacy.com/app/amawai/android-wikipedia-390/pullRequest?prid=1526985)